pkg/controller/installation/marketplace/manager.go

```diff
@@ -57,6 +57,9 @@ func (m *MarketplaceManager) CreateSubscription(os marketplacev1.OperatorSource,
 		ObjectMeta: metav1.ObjectMeta{
 			Namespace: ns,
 			Name:      pkg,
+			Labels: map[string]string{
+				"integreatly": "yes",
```

Comment:
We already use `integreatly-middleware-service=true` label in 3.x installation:
https://github.com/integr8ly/installation/search?q=integreatly-middleware-service&unscoped_q=integreatly-middleware-service
Maybe we should stick to that?